images/sqlpad/tests/main.tf

```diff
@@ -0,0 +1,32 @@
+terraform {
+  required_providers {
+    oci = { source = "chainguard-dev/oci" }
+  }
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+data "oci_exec_test" "manifest" {
+  digest      = var.digest
+  script      = "./02-curl-test.sh"
+  working_dir = path.module
+}
+
+data "oci_string" "ref" { input = var.digest }
+# Will work only when we use updated version postgresql 13.2.23 in upstream charts https://github.com/sqlpad/sqlpad/blob/f42ce1a84e1327e69178f38e7b3a2462dcaeeb7c/sqlpad-charts/requirements.yaml#L3
```

Comment:
I think I have already created an issue in their upstream, not sure an issue creation can help us here since I don't think it's a blocking problem? https://github.com/sqlpad/sqlpad/issues/1211